ansible/roles/test/files/ptftests/vxlan_traffic.py

```diff
@@ -59,6 +62,12 @@ def setUp(self):
         self.dut_mac = self.test_params['dut_mac']
         self.vxlan_port = self.test_params['vxlan_port']
         self.expect_encap_success = self.test_params['expect_encap_success']
+        self.packet_count = self.test_params['packet_count']
+        # The ECMP check fails occasionally if there is not enough packets.
+        # We should keep the packet count atleast 4.
+        if self.packet_count < 4:
+            print("WARN: packet_count is below minimum, resetting to 4")
+            self.packet_count = 4
 
         self.random_mac = "00:aa:bb:cc:dd:ee"
         self.ptf_mac_addrs = read_ptf_macs()
```

Comment:
Please add the code from this PR #5105
Lines 108, 115-120

Reply:
Will do.